### PR TITLE
BZ1263594: Creating Project, then copying it, then deleting it causes NoSuchFileException

### PR DESCRIPTION
**Pull request by manstis**
See https://bugzilla.redhat.com/show_bug.cgi?id=1263594

I also removed some old tests that (a) were not running, (b) have moved to ```kie-wb-common```

### CODE REVIEW

**Comment by manstis** [comment]
Related PR:

https://github.com/droolsjbpm/kie-wb-common/pull/110

**Comment by manstis** [comment]
master and 6.3.x please

File: guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/DeleteProjectObserverBridge.java [hunk 1]
```diff
@@ -34,20 +35,33 @@
...
 import org.uberfire.workbench.events.ResourceDeletedEvent;
 
+/**
+ * A bridge between changes made to an underlying VFS and Project abstractions. Projects can be deleted
+ * either through the Workbench's UI or via REST or on a cloned repository that's pushed back to the Workbench.
+ * Interested observers are signaled of a Project deletion event when a pom.xml file is detected as deleted.
+ */
 @ApplicationScoped
-public class PathObserverService {
+public class DeleteProjectObserverBridge {
```
Comment (Rikkola) [inline guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/DeleteProjectObserverBridge.java]: A lot better name. Good to have a class for single observation task, delete in this case.

**Comment by Rikkola** [comment]
guvnor
------
(master) http://github.com/droolsjbpm/guvnor/commit/fe742927e

(6.3.x)http://github.com/droolsjbpm/guvnor/commit/03ade8394

kie-wb-common
-------------
(master) http://github.com/droolsjbpm/kie-wb-common/commit/5e69ecdbf

(6.3.x) http://github.com/droolsjbpm/kie-wb-common/commit/1a6c74759

**Pull request closed without merging**